specs/_features/maxeb_increase/capella.py

```diff
@@ -1160,6 +1159,22 @@ def decrease_balance(state: BeaconState, index: ValidatorIndex, delta: Gwei) ->
     state.balances[index] = 0 if delta > state.balances[index] else state.balances[index] - delta
 
 
+def compute_exit_epoch_and_update_churn(state: BeaconState, exit_balance: Gwei) -> Epoch:
+    earliest_exit_epoch = compute_activation_exit_epoch(get_current_epoch(state))
+    if state.earliest_exit_epoch < earliest_exit_epoch
+        state.earliest_exit_epoch = earliest_exit_epoch
+        state.exit_balance_to_consume = get_validator_churn_limit(state)
+
+    if state.exit_balance_to_consume < exit_balance:
```

Comment:
This seems to make the assumption that `exit_balance` can't be large enough to require more than one extra epoch of churn

Reply:
Should use a `while` instead of `if`